apps/web/public/static/locales/en/common.json

```
@@ -2599,10 +2599,12 @@
   "outlook_connect_atom_loading_label": "Checking Outlook Calendar",
   "booking_question_response_variables": "Booking question response variables",
   "managed_by_teamAdmins": "Managed by {{teamAdmins}}",
-  "number_of_options":"{{count}} options",
+  "number_of_options": "{{count}} options",
   "reschedule_with_same_round_robin_host_title": "Reschedule with same Round-Robin host",
   "reschedule_with_same_round_robin_host_description": "Rescheduled events will be assigned to the same host as initially scheduled",
   "disable_input_if_prefilled": "Disable input if the URL identifier is prefilled",
   "you_are_unauthorized_to_make_this_change_to_the_booking": "You are unauthorized to make this change to the booking",
+  "show_optimized_slots": "Optimized slots",
+  "show_optimized_slots_description": "Instead of limiting events to our predefined start times (example - on the hour for 60-minute events), we find the optimal slot positioning to maximize the number of available slots.",
```

```suggestion
  "show_optimized_slots_description": "Find the optimal slot positioning to maximize the number of available slots.",
```
I'd say that's enough information 🤔